synapse/app/homeserver.py

```diff
@@ -194,6 +194,7 @@ def _configure_named_resource(
                 {
                     "/_matrix/client/api/v1": client_resource,
                     "/_matrix/client/r0": client_resource,
+                    "/_matrix/client/v1": client_resource,
```

Comment:
For the generic worker it seems we just put `client_resource` under `/_matrix/client` once. I'm not sure why we don't do that here too?

Reply:
I think it's because of `WEB_CLIENT_PREFIX`, down at line 274.

Reply:
(which is not to say there aren't way better ways of doing this and I would love it if this stuff were less of a mess. see also https://github.com/matrix-org/synapse/issues/5118)